dotnet/src/SemanticKernel/SkillDefinition/SKFunction.cs

```diff
@@ -361,6 +395,22 @@ private static async Task<string> GetCompletionsResultContentAsync(IReadOnlyList
         this.Name = functionName;
         this.SkillName = skillName;
         this.Description = description;
+        this.ExecutionTimeHistogram = s_meter.CreateHistogram<double>(
```

Comment:
@dmytrostruk Here I think it's important to include the skill and function, So I think we need to initialize it here, what do you think?